### PR TITLE
Fix: Return early

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] returns early

### CODE REVIEW

File: src/Element.php [hunk 1]
```diff
@@ -133,9 +133,13 @@ private function getNextNode($node, $checkChildren = true)
     {
         if ($checkChildren && $node->firstChild) {
             return $node->firstChild;
-        } elseif ($node->nextSibling) {
```
Comment (localheinz) [inline src/Element.php]: No need for `else` or `elseif` here if we return early already.

File: src/HtmlConverter.php [hunk 2]
```diff
@@ -105,9 +105,7 @@ public function convert($html)
         // Store the now-modified DOMDocument as a string
         $markdown = $document->saveHTML();
 
-        $markdown = $this->sanitize($markdown);
```
Comment (localheinz) [inline src/HtmlConverter.php]: We can just return the result of this expression directly.

File: src/HtmlConverter.php [hunk 3]
```diff
@@ -226,8 +224,6 @@ protected function sanitize($markdown)
             }
         }
 
-        $markdown = trim($markdown, "\n\r\0\x0B");
```
Comment (localheinz) [inline src/HtmlConverter.php]: We can just return the result of this expression directly.

File: src/Converter/ListItemConverter.php [hunk 4]
```diff
@@ -29,13 +29,12 @@ public function convert(ElementInterface $element)
         }
 
         if ($list_type === 'ul') {
-            $markdown = $prefix . '- ' . $value . "\n";
```
Comment (localheinz) [inline src/Converter/ListItemConverter.php]: We can just return early here.

File: src/Converter/ImageConverter.php [hunk 5]
```diff
@@ -19,12 +19,10 @@ public function convert(ElementInterface $element)
 
         if ($title !== '') {
             // No newlines added. <img> should be in a block-level element.
-            $markdown = '![' . $alt . '](' . $src . ' "' . $title . '")';
```
Comment (localheinz) [inline src/Converter/ImageConverter.php]: We can just return early here.

File: src/Converter/HeaderConverter.php [hunk 6]
```diff
@@ -36,9 +36,9 @@ public function convert(ElementInterface $element)
 
         if (($level === 1 || $level === 2) && !$element->isDescendantOf('blockquote') && $style === self::STYLE_SETEXT) {
             return $this->createSetextHeader($level, $element->getValue());
-        } else {
```
Comment (localheinz) [inline src/Converter/HeaderConverter.php]: No need for an `else` if we return already.

**Comment by colinodell** [comment]
Looks good :+1: Thanks!

**Comment by localheinz** [comment]
Thank you, @colinodell!

**Pull request merged**